front/app/components/PageForm/NavbarTitleField/index.tsx

```diff
@@ -28,7 +28,9 @@ const NavbarTitleField = ({
   intl: { formatMessage },
 }: Props & WrappedComponentProps) => {
   const { data: page } = useCustomPageById(pageId ?? undefined);
-  const navbarItem = useNavbarItem({ navbarItemId });
+  const { data: navbarItems } = useNavbarItems();
+
+  const navbarItem = navbarItems?.data.find((item) => item.id === navbarItemId);
```

Comment:
Why did you decide to not use the useNavbarItem hook anymore?

Reply:
All it was doing was filtering the data to find an item - I didn't feel we needed an abstracted separate hook for this.